tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -54,7 +54,7 @@ sub select_base_and_cleanup {
 sub run {
     my $self = shift;
 
-    if (!is_tumbleweed) {
+    if (!is_tumbleweed && !sle_version_at_least('15')) {
```

Comment:
Same as above for `is_sle` :)

Reply:
Thanks, I just fixed it, please help to review.